index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <script src="form.js" defer></script>
+    <script type="module" src="helpers.js"></script>
```

Comment:
Useless script all the necessary functions were imported to the form.js

Reply:
It doesn't work without type="module" when I tried to replace fetch with axios.

Reply:
Useless scripts. index.html shouldn't have script tags at all. All the scripts will be injected by webpack.